1-exercises/B-highlight-words/script.js

```diff
@@ -1,5 +1,41 @@
 function highlightWords(paragraph, colours) {
   // Write your code here...
+
+  const p = document.createElement("p");
+
+  const selectColor = document.createElement("select");
+
+  colours.forEach((color) => {
+    const colorOptions = document.createElement("option");
+    colorOptions.value = color;
+    colorOptions.textContent = color.toUpperCase();
```

Comment:
This one is a little tricky: `.toUpperCase()` upper-cases an entire string, rather than just the first character.

You could do something like:

```suggestion
    colorOptions.textContent = `${color[0].toUpperCase()}${color.slice(1)}`;
```

or make a separate function to do this.